doc/conf.py

```diff
@@ -25,7 +25,7 @@
 templates_path = ['_templates']
 source_suffix = '.rst'
 source_encoding = 'utf-8-sig'
-master_doc = 'index'
+root_doc = 'index'
 
```

Comment:
This is ok, but maybe add a lil description in your commit (changed in version 4.0 of Sphynx)

Reply:
I'll fix it in final commit rewrite. (I think otherwise it messes up what changes you have already looked up).

Reply:
Ok, i had to do a rebase anyway and edited the commit message. (However, i'll do a final clean up later nonetheless).